src/selenium/roomPage.js

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable no-empty-label */
+'use strict';
+
+const BasePage = require('./basePage.js');
+const By = require('selenium-webdriver').By;
+
+const RoomPage = Object.create(BasePage);
+
+RoomPage.checkIsolatedBookmark = function() {
+  const self = this;
+  const bookmarkSessionsIdsArr = ['3014'];
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 2.](https://app.codacy.com/manual/triplez/open-event-webapp/pullRequest?prid=3226522)